db/data/regions/CL/fr.yml

```diff
@@ -0,0 +1,26 @@
+---
+fr:
+  worldwide:
+    CL:
+      addresses:
+        neighborhood:
+          label:
+            default: Commune
+            optional: Commune (facultatif)
+          errors:
+            blank: Saisir une commune
+            too_long: La commune est trop long (le maximum est de 255 caractères)
```

Comment:
This should be _La commune est trop **longue** (le maximum est de 255 caractères)_.